05-cond.md

```diff
@@ -86,7 +86,7 @@ We can also combine tests using `and` and `or`.
 `and` is only true if both parts are true:
 
 ~~~ {.python}
-if (1 > 0) and (-1 > 0):
+if 1 > 0 and -1 > 0:
```

Comment:
Pedantically, I think that we should avoid unnecessary operators because we're implicitly teaching that they *are* necessary. Practically, there are many languages that require parentheses in if statements (`if (expression)`) and so I want to reinforce that this isn't necessary  (and, to me, looks really weird) in Python.

But, I don't feel that strongly about it so I'm happy to add them back in; anyone case to break the tie here?